tutorial/setup/server-management.md

```diff
@@ -0,0 +1,21 @@
+#Basic Server User Management
```

Comment:
I assume you're aware / still working on the fact that it is not in the sidebar listing.

Reply:
Well i'm not aware at least :/

Reply:
Those nav files I referenced in the discord are the place to edit so that it appears :)

Reply:
I do see there's a file in docs/management.md that's referenced in the general self-hosted page. Gonna move it there so we don't have to make redundant copies save you a few bytes of disk space :) that said not sure that file belong in the dm folder since it pertains to the server. Assuming we keep that reference.

Reply:
It should indeed ideally move to a Server section or such

Reply:
Just pushed an update, see if it works

Reply:
ok I'm beat on this. Any idea why it's failing to build here? keep getting
> YAMLException: end of the stream or a document separator is expected at line 50, column 1:

Reply:
actually wait. did I really forget a damn spce? darnit

Reply:
:D happens to the best of us

Reply:
no worries I forgot two of them maybe it builds right this time

Reply:
It built right this time around :D now to get syntax highlighting to work